projects/1.20/assets/mowzies-mobs/mowziesmobs/lang/zh_cn.json

```
@@ -0,0 +1,402 @@
+  "advancements.mowziesmobs.kill_frostmaw.title": "野蛮觉醒",
+  "advancements.mowziesmobs.kill_frostmaw.description": "击杀霜冻巨兽",
+  "advancements.mowziesmobs.kill_naga.title": "飞龙在天",
+  "advancements.mowziesmobs.kill_naga.description": "击杀娜迦",
+  "advancements.mowziesmobs.kill_grottol.title": "你简直...",
+  "advancements.mowziesmobs.kill_grottol.description": "击杀岩壳居蟹",
+  "advancements.mowziesmobs.sneak_grove.title": "一树之鸟",
+  "advancements.mowziesmobs.sneak_grove.description": "在伪装状态下潜入烈阳鸟巢穴",
+  "advancements.mowziesmobs.suns_blessing.title": "太阳法师",
+  "advancements.mowziesmobs.suns_blessing.description": "接受太阳祝福",
+  "advancements.mowziesmobs.steal_ice_crystal.title": "没头脑时间",
+  "advancements.mowziesmobs.steal_ice_crystal.description": "从霜冻巨兽处偷走冰霜结晶",
+  "advancements.mowziesmobs.kill_grottol_fortune.title": "搏一搏! ",
+  "advancements.mowziesmobs.kill_grottol_fortune.description": "使用附有时运魔咒的镐击杀岩壳",
+  "advancements.mowziesmobs.kill_grottol_silk_touch.title": "跟我走吧",
+  "advancements.mowziesmobs.kill_grottol_silk_touch.description": "使用附有精准采集魔咒的镐击杀岩壳",
+
+  "subtitles.mowziesmobs.foliaath.bite": "巨噬叶: 啃咬",
+  "subtitles.mowziesmobs.foliaath.pant": "巨噬叶: 喘息",
+  "subtitles.mowziesmobs.foliaath.grunt": "巨噬叶: 咕哝声",
+  "subtitles.mowziesmobs.foliaath.emerge": "巨噬叶: 出现",
+  "subtitles.mowziesmobs.foliaath.retreat": "巨噬叶: 缩回",
+  "subtitles.mowziesmobs.foliaath.die": "巨噬叶: 死亡",
+  "subtitles.mowziesmobs.foliaath.hurt": "巨噬叶: 受伤",
+  "subtitles.mowziesmobs.foliaath.rustle": "巨噬叶: 沙沙声",
+  "subtitles.mowziesmobs.foliaath.baby_eat": "巨噬叶幼苗: 吞吃",
+  "subtitles.mowziesmobs.wroughtnaut.whoosh": "千锻巨斧: 嗖嗖声",
+  "subtitles.mowziesmobs.wroughtnaut.creak": "钢铁守护者: 吱呀作响",
+  "subtitles.mowziesmobs.wroughtnaut.grunt": "钢铁守护者: 咕哝声",
+  "subtitles.mowziesmobs.wroughtnaut.hurt": "钢铁守护者: 受伤",
+  "subtitles.mowziesmobs.wroughtnaut.pre_swing": "钢铁守护者: 起手",
+  "subtitles.mowziesmobs.wroughtnaut.pull": "钢铁守护者: 拔斧子",
+  "subtitles.mowziesmobs.wroughtnaut.release": "钢铁守护者: 拔出斧子",
+  "subtitles.mowziesmobs.wroughtnaut.shout": "钢铁守护者: 尖啸",
+  "subtitles.mowziesmobs.wroughtnaut.scream": "钢铁守护者: 死亡",
+  "subtitles.mowziesmobs.wroughtnaut.swing": "钢铁守护者: 攻击",
+  "subtitles.mowziesmobs.wroughtnaut.footstep": "钢铁守护者: 迈步",
+  "subtitles.mowziesmobs.wroughtnaut.axe_clang": "千锻巨斧: 铿锵声",
+  "subtitles.mowziesmobs.umvuthana.shout": "烈阳鸟: 喊叫",
+  "subtitles.mowziesmobs.umvuthana.battlecry": "烈阳鸟: 战吼",
+  "subtitles.mowziesmobs.umvuthana.die": "烈阳鸟: 死亡",
+  "subtitles.mowziesmobs.umvuthana.hurt": "烈阳鸟: 咕哝声",
+  "subtitles.mowziesmobs.umvuthana.idle": "烈阳鸟: 啼叫",
+  "subtitles.mowziesmobs.umvuthana.alert": "烈阳鸟: 啼叫",
+  "subtitles.mowziesmobs.umvuthana.roar": "烈阳鸟: 尖叫",
+  "subtitles.mowziesmobs.umvuthana.rattle": "烈阳鸟: 吱呀作响",
+  "subtitles.mowziesmobs.umvuthana.attack": "烈阳鸟: 攻击",
+  "subtitles.mowziesmobs.umvuthana.attack_big": "烈阳鸟: 攻击",
+  "subtitles.mowziesmobs.umvuthana.emerge": "烈阳鸟: 出现",
+  "subtitles.mowziesmobs.umvuthana.retract": "烈阳鸟: 回收",

```

| **审阅级别** |
|:--------------:|
|❓ **疑问** / 不阻止合并|

原文：**Umvuthana retracts**
```suggestion
  "subtitles.mowziesmobs.umvuthana.retract": "烈阳鸟: 回收",
```
这是什么情景的声音

---

佩戴太阳面具时对召唤出的烈阳鸟点击鼠标右键回收的音效